projects/account/src/app/modules/profile/pages/verify-email/verify-email.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { faEnvelopeCircleCheck, IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import { environment } from '@account/environments/environment';
+import { ProfileService } from '@account/http/profile.service';
+
+@Component({
+  selector: 'account-verify-email',
+  templateUrl: './verify-email.component.html',
+  styleUrls: ['./verify-email.component.scss']
+})
+export class VerifyEmailComponent implements OnInit {
+    public emailAddressIcon: IconDefinition = faEnvelopeCircleCheck;
+    public emailVerified: boolean;
+    public verificationMessage: string;
+
+    constructor(
+        private route: ActivatedRoute,
+        private profileService: ProfileService,
+        private router: Router
+    ) { }
+
+    ngOnInit(): void {
+        const resetToken = this.route.snapshot.queryParams['token'];
+        this.profileService.verifyEmailAddress(resetToken).subscribe({
+            next: () => {
+                this.verificationMessage = 'Thank you for verifying your new email address.  Use it the next time you login.';
```

Comment:
Are we verifying the address on account creation too? 
If so might remove the word "new".

Reply:
This is just for the email verification step when changing the email address.